### ISSUE TITLE
YOUTUBE MUSIC BUFFER

### ISSUE THREAD

**Issue by Hitesh666** [StarsWarrior/BlackHole#16]
App can't play YouTube music in last 2 days .
I think app can't extract (separate) mp3 file

**Comment by AhmedJu5002** [comment]
Yeah, same problem... Can't play music from Yt-music and YouTube section...
I thought reinstalling the app may be fix it.. but it now I can't log in into Spotify also😭😭..
@Sangwan5688 where are you??😭

**Comment by kametepe** [comment]
Hello
the plugin in charge of extracting Youtube music has to be updated .
Please use **youtube_explode_dart 2.2.2**
Regards

**Comment by kametepe** [comment]
> Hey @kametepe now there is only one problem which is Spotify integration problem... Can you solve it too?? Please

Hello 
Do you have a specific error message regarding Spotify?
I can debug but wanna know first.
Regards

**Comment by AhmedJu5002** [comment]
When I try to sing in to Spotify from black hole.. my browser opens up and show me this
![Uploading Screenshot_20241007-220346.png…]()

**Comment by AhmedJu5002** [comment]
Maybe there is some client_id issue...
Please check Mr. @kametepe
